pinot-spi/src/main/java/org/apache/pinot/spi/env/CommonsConfigurationUtils.java

```diff
@@ -288,6 +195,7 @@ public static <T> T convert(Object value, Class<T> returnType) {
    * - Escaping comma with backslash doesn't work when comma is preceded by a backslash
    */
   public static String replaceSpecialCharacterInPropertyValue(String value) {
+    value = StringEscapeUtils.escapeJava(value);
```

Comment:
Is this a bugfix?

Reply:
This is needed in commons-configuration2, as it does not support string escape/unescape like commons-configuration1.

Reply:
Can you give an example of what the property will look like in v1 vs v2? Will v1 automatically escape/unescape the value?